### ISSUE TITLE
All builds stuck until exceeding time limit

### ISSUE THREAD

**Issue by UpcraftLP** [josephbmanley/build-godot-action#28]
I've tried setting up this action yesterday, but it seems like nothing ever happens.
There is no console log, and after 6 hours github eventually cancelled all my builds.

The project is fairly simple, so that shouldnt be the issue (however note it does use C#/.NET).
<br>
If you need more details, you can find the failed run [here](https://github.com/Beak-A-Boo/FlightSim/actions/runs/8430281541).

<br>

![image](https://github.com/yeslayla/build-godot-action/assets/10714589/5691e3b1-fa51-4861-8c5f-5116b3d456dd)

**Comment by Calinou** [comment]
It looks like the editor is running in headless mode while doing nothing. Remember to use the `--export` command line argument if you want the project to be exported – the editor will exit automtaically in this case.

The command line should look similar to this:

```
path/to/godot.binary --headless --export-release "Windows Desktop" path/to/output.exe
```

**Comment by UpcraftLP** [comment]
> It looks like the editor is running in headless mode while doing nothing. Remember to use the `--export` command line argument if you want the project to be exported – the editor will exit automtaically in this case.
> 
> The command line should look similar to this:
> 
> ```
> path/to/godot.binary --headless --export "Windows Desktop" path/to/output.exe
> ```

The action seems to be using [`--export-release`](https://github.com/yeslayla/build-godot-action/blob/f362e14dc5d8290ac81ee3ed7c799242dd3215af/entrypoint.sh#L14)

**Comment by Calinou** [comment]
> The action seems to be using [--export-release](https://github.com/yeslayla/build-godot-action/blob/f362e14dc5d8290ac81ee3ed7c799242dd3215af/entrypoint.sh#L14)

This is correct (`--export` is a legacy option, which acted the same as `--export-release`). I've edited my comment accordingly.

**Comment by UpcraftLP** [comment]
so.. it is doing exactly what you are suggesting already?

**Comment by Calinou** [comment]
> so.. it is doing exactly what you are suggesting already?

Yes :slightly_smiling_face: 

Try adding `--quit-after 100` in the command line. This quits after 100 process iterations (which should only happen after the project is done exporting). That said, `--export` is supposed to have the editor close itself automatically once the project is done exporting.

**Comment by UpcraftLP** [comment]
but I dont have access to the process's command line..
